tests/conftest.py

```diff
@@ -65,7 +65,7 @@ def _init_mock_data(s3_client, bucket, study, data_package, version):
     )
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
one wierd trick to eliminate all those OS env var imports, coming to other repos near you shortly.